### PR TITLE
Complete UI Screens without responsiveness

### PR DESCRIPTION
**Pull request by Muaviya-ShayanSolutions**

### CODE REVIEW

File: src/components/card/Card.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }, []);
+
+  const navigate = useNavigate();
+  const navigateToContacts = () => {
+    navigate("/country-detail");
+  };
+  
+  const test = countriesData.map((country,index) => {
+    
+    console.log(country.name.common);
```
Comment (imran-munawar) [inline src/components/card/Card.js]: Remove all the console.log

File: src/components/navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React,{useState} from "react";
+import "../.././index.css";
+import "./navbarStyle.css";
+const Navbar = () => {
+  const [mode, setMode] = useState(() => {
+    return "light";
+  }); 
+  const toggleMode = () => {
+    console.log(mode)
```
Comment (imran-munawar) [inline src/components/navbar/Navbar.js]: Remove log

File: src/components/card/Card.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  useEffect(() => {
+    fetchUserData();
+  }, []);
+
+  const navigate = useNavigate();
+  const navigateToContacts = () => {
+    navigate("/country-detail");
+  };
+  
+  const test = countriesData.map((country,index) => {
```
Comment (imran-munawar) [inline src/components/card/Card.js]: the test is not a perfect naming convention. You can also directly map countries data in return of this component.

File: src/components/card/Card.js (hunk 1 above)
Comment (Muaviya-ShayanSolutions) [reply src/components/card/Card.js]: ok

**Pull request merged**